src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public String getStartMessage() {
+        return "Bulldozer start working";
+    }
+
+    public String getStopMessage() {
+        return "Bulldozer stop working";
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(this.getClass().getName() + ": " + getStartMessage());

```

Comment:
`this.getClass().getName()` print - > core.basesyntax.Bulldozer, so if you would like to use it getSimpleName() is better

But, you can just use
```suggestion
        System.out.println("Bulldozer start working");
```

The same for other methods and other machines